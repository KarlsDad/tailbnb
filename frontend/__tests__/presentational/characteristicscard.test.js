import React from 'react';
import { shallow } from 'enzyme';

// Components
import CharactersiticCard from '../../src/components/presentational/CharacteristicCard';
import { isType } from 'graphql';

describe('Characteristic Card', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<CharactersiticCard />);
    expect(component).toMatchSnapshot();
  });
});
