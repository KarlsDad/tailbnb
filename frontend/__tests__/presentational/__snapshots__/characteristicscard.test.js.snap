// Jest Snapshot v1, https://goo.gl/fbAQLP

<<<<<<< HEAD
exports[`Characteristic Card should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
=======
exports[` 1`] = `ShallowWrapper {}`;

exports[`Characteristics Card should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
>>>>>>> 9cb84468960711922a5d55a444848face912a73b
